client/components/date-range/style.scss

```diff
@@ -0,0 +1,129 @@
+.date-range {
+	position: relative;
+	display: flex;
+	align-items: baseline;
+}
+
+
+.date-range__trigger-btn {
+	text-align: left; // required to avoid content "jumping" as date text changes
+
+	.toggle-visible & {
+		box-shadow: none;
+		background-color: var(--color-neutral-0);
+	}
+
+	.gridicons-chevron-down {
+		fill: var(--color-neutral-500);
+		margin: 0 0 0 5px;
+		flex-shrink: 0;
+		transition: transform 0.15s cubic-bezier( 0.175, 0.885, 0.32, 1.275 );
+
+		.is-disabled & {
+			fill: var(--color-neutral-50);
+		}
+
+		.toggle-visible & {
+			transform: rotate( -180deg );
+		}
+	}
+}
+
+
+.date-range__trigger-btn-text {
+	width: 12.5em; // required to avoid content "jumping" as date text changes
```

Comment:
This change tests well and I think I do prefer this over using a monospace font (we shouldn't be forced to select a font based on this problem).
What do you think about changing this to `min-width` though? I noticed that changing the characters to be wider manually resulted in a line break. Of course these actual values couldn't exist, but there may be some valid dates that are equally as wide:

<img width="222" alt="screenshot 2019-01-14 at 15 14 03" src="https://user-images.githubusercontent.com/4335450/51121480-794a4f80-180f-11e9-9a21-eef463611836.png">

Reply:
> This change tests well and I think I do prefer this over using a monospace font (we shouldn't be forced to select a font based on this problem).

👍 

> What do you think about changing this to min-width though? 

Yes `min-width` would be safer just in case some random value we haven't tested causes it to expand.

>  I noticed that changing the characters to be wider manually resulted in a line break

I was trying to find as many horrible combinations as possible - I knew I'd miss one! Thanks for being so thorough. I'll tweak again slightly wider and then move to `min-width`

Reply:
✅ Done